gadget-container/gadgets/collector-process/c/ebpf.c

```diff
@@ -0,0 +1,112 @@
+#include <linux/bpf.h>
```

Comment:
What about namign this file in `gadget-container/gadgets/collector-process/bpf/collector-process.c` instead?

Reply:
Done.